src/test/java/com/wikia/webdriver/common/testnglisteners/BrowserAndTestEventListener.java

```diff
@@ -53,7 +54,19 @@ public void afterNavigateTo(String url, WebDriver driver) {
     if (method != null) {
       Class<?> declaringClass = method.getDeclaringClass();
 
-      String cookieDomain = String.format(".%s", Configuration.getEnvType().getDomain());
+      // Fallback to wikia.com domain
+      String cookieDomain = Configuration.getEnvType().getWikiaDomain();
+
+      for(EnvType env: EnvType.values()){
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS135)